installation fails

I am getting the error below while installing adhocracy3
```
2017-01-15 16:26:54,524 WARNI [substanced.catalog.subscribers][MainThread] Cannot autosync/autoreindex catalog due to unfinished evolve steps
Updating resources.
resources: Running . /home/vagrant/Dev/adhocracy3/source_env
git ls-files --other src/mercator/mercator/build/js/Resources_ | xargs rm -f
/home/vagrant/Dev/adhocracy3/bin/tsc --module commonjs --pretty src/mercator/mercator/build/js/mkResources.ts
/home/vagrant/Dev/adhocracy3/bin/node src/mercator/mercator/build/js/mkResources.js src/mercator/mercator/build/meta_api.json src/mercator/mercator/build/js
Installing javascript.
javascript: Running '/home/vagrant/Dev/adhocracy3/bin/node /home/vagrant/Dev/adhocracy3/bin/tsc -m umd --sourcemap --pretty src/mercator/mercator/build/js/Adhocracy*.ts'

245                 financialPlanUrl = this.adhConfig.financial_plan_url_en;
                                                      ~~~~~~~~~~~~~~~~~~~~~

src/mercator/mercator/build/js/Packages/Mercator/2015/Proposal/Proposal.ts(245,51): error TS2339: Property 'financial_plan_url_en' does not exist on type 'IService'.
```

--- a/src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Config/Config.ts
+++ b/src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/Config/Config.ts
@@ -29,7 +29,8 @@ export interface IService {
     embed_only : boolean;
     hide_header : boolean;
     mercator_platform_path: string;
-    financial_plan_url_de: string;
+    financial_plan_url_de?: string;
+    financial_plan_url_en?: string;
     site_name : string;
     netiquette_url : string;
     cachebust : boolean;

--- a/src/mercator/mercator/defaults.yaml
+++ b/src/mercator/mercator/defaults.yaml
@@ -7,3 +7,4 @@ adhocracy:
       de: 'https://advocate-europe.eu/de/terms-of-use'
   mercator_platform_path: '/mercator/'
   financial_plan_url_de: ''
+  financial_plan_url_en: ''

